packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -220,5 +220,7 @@ export const messages = {
   error_push_or_deploy_on_save_no_default_username:
     'Error running push or deploy on save: No default org is set. Run "SFDX: Create a Default Scratch Org" or "SFDX: Authorize an Org", then push or deploy the changes that you just saved. Or, to disable push or deploy on save, set "salesforcedx-vscode-core.push-or-deploy-on-save.enabled" to false in your user or workspace settings for VS Code.',
   error_source_path_not_in_package_directory_text:
-    'Error deploying or retrieving source: The file or directory that you tried to deploy or retrieve isn\'t in a package directory that\'s specified in your sfdx-project.json file. Add this location to your "packageDirectories" value, or deploy or retrieve a different file or directory. For details about sfdx-project.json, see: https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_ws_config.htm'
+    'Error deploying or retrieving source: The file or directory that you tried to deploy or retrieve isn\'t in a package directory that\'s specified in your sfdx-project.json file. Add this location to your "packageDirectories" value, or deploy or retrieve a different file or directory. For details about sfdx-project.json, see: https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_ws_config.htm',
+  error_parsing_sfdx_project_file:
+    'Sfdx project file seems invalid. Unable to parse %s. %s'
```

Comment:
Hi, @ruthemmanuelle. I copied this text from an error message that I saw coming from the LWC extension that reads: 
![screen shot 2019-01-30 at 11 17 44 am](https://user-images.githubusercontent.com/1639375/52007381-93309700-248b-11e9-81ac-cd724518df4f.png)

The error message that I'm adding reads:
![screen shot 2019-01-30 at 11 17 08 am](https://user-images.githubusercontent.com/1639375/52007406-9f1c5900-248b-11e9-9775-8281e4a65378.png)

Reply:
Please let me know what you think.

Reply:
It's a bit long, and "seems invalid" feels kind of wishy-washy. How about this?

`Couldn't parse sfdx-project.json file (%s). Parse error: %s`